third_party/rust/chromium_crates_io/Cargo.lock

```diff
@@ -2876,6 +2878,23 @@ dependencies = [
  "blake2b_simd",
 ]
 
+[[package]]
+name = "zerocopy"
```

Comment:
adding new dependencies requires 3p code review, can you open a ticket for these?

Reply:
oh wait, are these upstream deps? If they've already gone through upstream review then they're ok

Reply:
cc: @cdesouza-chromium

Reply:
nvm, these are all upstream